tests/Fixtures/ci-integration/setUp.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
this file was created when i was trying to test this whole behaviour in bash only (with `bats` - tests runner), it requires a setup to be dedicated file
with current php-based `executeCommand` method we could probably run setup with that method instead of dedicated file.
That's what you mean?